static/app/styles/global.tsx

```diff
@@ -34,10 +34,17 @@ const styles = (theme: Theme, isDark: boolean) => css`
 
   pre,
   code {
-    background-color: ${theme.backgroundSecondary};
     color: ${theme.textColor};
   }
 
+  pre {
+    background-color: ${theme.backgroundSecondary};
+  }
+
+  code {
+    background-color: transparent;
+  }
```

Comment:
hmm do we need this block? shouldn't `code` have transparent backgrounds by default?